spec/timezone.html

```diff
@@ -3,5 +3,240 @@
+
+    <emu-clause id="sec-temporal.timezone.prototype.getdatetimefor">
+      <h1>Temporal.TimeZone.prototype.getDateTimeFor ( _absolute_ )</h1>
+      <p>
+        The `getDateTimeFor` method takes one argument _absolute_.
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _timeZone_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_timeZone_, [[InitializedTemporalTimeZone]]).
+        1. Let _absolute_ be ? ToAbsolute(_absolute_).
+        1. Return ? GetDateTimeFor(_timeZone_, _absolute_);
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal.timezone.prototype.getabsolutefor">
+      <h1>Temporal.TimeZone.prototype.getAbsoluteFor ( _dateTime_ [ , _disambiguation_ ] )</h1>
+      <p>
+        The `getAbsoluteFor` method takes two arguments, _dateTime_ and _disambiguation_.
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _timeZone_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_timeZone_, [[InitializedTemporalTimeZone]]).
+        1. <mark>TODO</mark>
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal.timezone.prototype.gettransitions">
+      <h1>Temporal.TimeZone.prototype.getTransitions ( _startingPoint_ )</h1>
+      <p>
+        The `getTransitions` method takes one argument _startingPoint_.
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _timeZone_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_timeZone_, [[InitializedTemporalTimeZone]]).
+        1. <mark>TODO</mark>
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal.timezone.prototype.tostring">
+      <h1>Temporal.TimeZone.prototype.toString ( )</h1>
+      <p>
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _timeZone_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_timeZone_, [[InitializedTemporalTimeZone]]).
+        1. Return _timeZone_.[[Identifier]].
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal.timezone.prototype.tojson">
+      <h1>Temporal.TimeZone.prototype.toJSON ( )</h1>
+      <p>
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _timeZone_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_timeZone_, [[InitializedTemporalTimeZone]]).
+        1. Return _timeZone_.[[Identifier]].
+      </emu-alg>
+    </emu-clause>
+  </emu-clause>
+
+  <emu-clause id="sec-temporal-timezone-abstract-ops">
+    <h1>Abstract operations</h1>
+
+    <emu-clause id="sec-temporal-timezonefromstring" aoid="TimeZoneFromString">
+      <h1>TimeZoneFromString ( _timeZone_ )</h1>
+      <emu-alg>
+        1. Assert: Type(_timeZone_) is String.
+        1. Let _valid_ be *true* if _timeZone_ matches <a href="https://github.com/tc39/proposal-temporal/blob/main/polyfill/lib/regex.mjs">the "timezone" regular expression in the polyfill</a> and *false* otherwise.
```

Comment:
same comment here re regex notation and alternatives (Not sure if the todo after this line, applies to it)